### PR TITLE
Updating Feature Case Studies

### PR DESCRIPTION
**Pull request by kbarnard10**
Updating order

### CODE REVIEW

**Comment by alexcontini** [comment]
@KimMcMahon

**Comment by zacharysarah** [comment]
@alexcontini 👋 It looks like we need to give you more ownership permissions.

### TLDR

You'll be able to approve this PR shortly.

### Short version

I'm opening a PR to add you as an OWNER to `/resources/`. The additional permissions will let you approve changes to CSS/SASS as well as to case studies. 

### Longer version

I was confused why your `/approve` didn't work on this PR, as it touches files in `/content/en/case-studies/`&mdash;until I realized the PR contains changes to files in `/resources/`, which you don't currently have permissions to change. 

Because so much of the case studies work involves updates to CSS and SASS, I think it makes sense to expand your approver status to include resource files as well as content in `/content/en/case-studies/`. 

I'll open a PR to add an `OWNERS` file in `/resources/` that inherits approver permissions from the top-level OWNERS file, and adds you as an individual approver for any files in the `/resources/` directory.

/cc @kbarnard10 👋

**Comment by alexcontini** [comment]
@zacharysarah that's so great, thank you very much I really appreciate it!

**Comment by kbarnard10** [comment]
@alexcontini You should be able to merge this PR now by adding:
/lgtm
/approve
